inc/Engine/CDN/RocketCDN/NoticesSubscriber.php

```diff
@@ -353,4 +354,60 @@ public function purge_cache_notice() {
 	private function is_white_label_account() {
 		return (bool) rocket_get_constant( 'WP_ROCKET_WHITE_LABEL_ACCOUNT' );
 	}
+
+	/**
+	 * Change CName admin notice contents.
+	 *
+	 * @return void
+	 */
+	public function change_cname_notice() {
+		if ( ! current_user_can( 'rocket_manage_options' ) ) {
+			return;
+		}
+
+		if ( 'settings_page_wprocket' !== get_current_screen()->id ) {
+			return;
+		}
+
+		$boxes = get_user_meta( get_current_user_id(), 'rocket_boxes', true );
+
+		if ( in_array( 'rocketcdn_change_cname', (array) $boxes, true ) ) {
+			return;
+		}
+
+		$old_cname = get_rocket_option( 'rocketcdn_old_url' );
```

Comment:
This would get the option from the `wp_rocket_settings` array, I don't think that's what you want

Reply:
That's what I wanted, I put this `rocketcdn_old_url` inside wp_rocket_settings main one.